src/adaptations/device-layer/include/Weave/DeviceLayer/ConfigurationManager.h

```diff
@@ -111,6 +123,11 @@ class ConfigurationManager
     bool IsPairedToAccount();
     bool IsMemberOfFabric();
     bool IsFullyProvisioned();
+    bool DeviceCredentialsProvisioned();
+    bool DeviceICACertsProvisioned();
+    bool ManufAttestDeviceICACertsProvisioned();
```

Comment:
I don't think we need DeviceICACertsProvisioned() and ManufAttestDeviceICACertsProvisioned() as part of this API.  I'm not sure we need them at all.